### PR TITLE
Data Exfilteration related changes

### PR DESCRIPTION
**Pull request by abhishekbhol**
added managedVirtualNetworkSettings definition.

<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i>

### Contribution checklist:
- [x] I commit to follow the [Breaking Change Policy](http://aka.ms/bcforapi) of “no breaking changes
- [x] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [x] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and errors have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist
- [ ] Ensure to check this box if one of the following scenarios meet updates in the PR, so that label “WaitForARMFeedback” will be added automatically to involve ARM API Review. Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs, all “removals” and “adding a new property” no more require ARM API review.
  - Adding new API(s)
  - Adding a new API version
  - Adding a new service

- [x] If you are blocked on ARM review and want to get the PR merged with urgency, please get the ARM oncall for reviews (*RP Manifest Approvers* team under <ins>Azure Resource Manager service</ins>) from IcM and reach out to them. 

### Breaking Change Review Checklist 
If there are following updates in the PR, ensure to request an approval from API Review Board as defined in the [Breaking Change Policy](http://aka.ms/bcforapi). 

- [ ] Removing API(s) in stable version
- [ ] Removing properties in stable version
- [ ] Removing API version(s) in stable version
- [ ] Updating API in stable version with Breaking Change Validation errors
- [ ] Updating API(s) in preview over 1 year

Please follow the link to find more details on [PR review process](https://aka.ms/SwaggerPRReview).

### CODE REVIEW

**Review by idear1203 — APPROVED** [review]

**Comment by abhishekbhol** [comment]
Do not Merge. Need a confirm few things

**Comment by chiragg4u** [comment]
> Do not Merge. Need a confirm few things

Added DonotMerge tag

**Comment by chiragg4u** [comment]
As identified in the automated checks there are breaking changes, Please follow Azure Breaking changes approval request form as defined @ [Breaking Change Process](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37673/Retirements-and-Breaking-Changes)

**Comment by abhishekbhol** [comment]
Hi @chiragg4u  I have verified that the changes are safe. Please remove the Do not merge tag and merge this PR.

**Comment by abhishekbhol** [comment]
Verified with @idear1203  these addition of new property is not a breaking change for Synapse. Please take this PR to completion as some features are blocked on this.

**Comment by erich-wang** [comment]
Adding new property in response is considered as breaking change, please check https://github.com/Azure/openapi-diff/blob/master/docs/rules/1041.md.
However, considering this API version is in preview and less than one year, breaking change is allowed if you service team sign off the breaking change.

@abhishekbhol , please confirm service team wants to merge this breaking change and understands the impact to existing customers, then I'll merge  this PR, thanks.

**Comment by abhishekbhol** [comment]
Hi @erich-wang , I have confirmed RP Synapse wants to merge this change. Please go ahead and merge it. Thanks

File: specification/synapse/resource-manager/Microsoft.Synapse/preview/2019-06-01-preview/workspace.json [hunk 1]
```diff
@@ -815,6 +841,28 @@
           "description": "The identity of the workspace",
           "$ref": "#/definitions/ManagedIdentity"
         },
+        "managedVirtualNetworkSettings": {
+          "description": "Managed Virtual Network Settings",
+          "type": "object",
+          "properties": {
+            "preventDataExfiltration": {
```
Comment (aim-for-better) [inline specification/synapse/resource-manager/Microsoft.Synapse/preview/2019-06-01-preview/workspace.json:848]: Is there a typo here, should it be preventDataExfilt**e**ration?

Comment (idear1203) [reply specification/synapse/resource-manager/Microsoft.Synapse/preview/2019-06-01-preview/workspace.json:848]: No, I don't think it is a typo. It should be `data exfiltration`

**Pull request merged**